packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummaryFragments.js

```diff
@@ -1,7 +1,7 @@
 import gql from 'graphql-tag';
 
 import { DiscountSummaryFragment } from './discountSummary';
-import { GiftCardSummaryFragment } from './giftCardSummary';
+import { GiftCardSummaryFragment } from './queries/giftCardSummary';
```

Comment:
@revanth0212 If I change import to `./queries/GiftCardSummary.ee` AWS builds pass. Tested in https://github.com/magento/pwa-studio/pull/2138.  Do we need to define any env variable?

Reply:
Found the issue @dpatil-magento 

We are referencing `queries/giftCardSummary` but the file name is `queries/GiftCardSummary`. Mac does not throw an error if the file name case is wrong, but Linux machines do. Git did not recognize the file name case change and did not push the changes. Hence it is working in our locals but fails in AWS.

I will be pushing the changes soon.